src/__tests__/int/consumer.int.spec.ts

```diff
@@ -1,10 +1,20 @@
-import { Subscribe } from "../..";
+import { Publish, Subscribe } from "../..";
 jest.unmock("kafkajs");
 
 describe("Consumer Integration Tests", () => {
-  const propertiesResponse = ["key", "message", "offset", "partition", "topic"];
+  beforeAll(async () => {
```

Comment:
You should add an afterAll method that clears the queue, due to when the test fails, the queue stays empty